RoslynPluginGenerator/NuGetPackageHandler.cs

```diff
@@ -58,9 +68,9 @@ public FileConflictResolution ResolveFileConflict(string message)
             }
         }
 
-        public NuGetPackageHandler(string packageSource, Common.ILogger logger)
+        public NuGetPackageHandler(Common.ILogger logger, string remotePackageSource, string localPackageDestination = null)
```

Comment:
Defaults for optional parameters must be compile-time constant, so it's null by default and assigned below.